app/scripts/modules/amazon/src/loadBalancer/configure/classic/HealthCheck.tsx

```diff
@@ -1,24 +1,27 @@
 import * as React from 'react';
-import { Field, FormikProps } from 'formik';
+import { Field, FormikErrors } from 'formik';
 
 import { IWizardPageProps, wizardPage } from '@spinnaker/core';
 
 import { IAmazonClassicLoadBalancerUpsertCommand } from 'amazon/domain';
 
-class HealthCheckImpl extends React.Component<IWizardPageProps & FormikProps<IAmazonClassicLoadBalancerUpsertCommand>> {
+export type IHealthCheckProps = IWizardPageProps<IAmazonClassicLoadBalancerUpsertCommand>;
+
+class HealthCheckImpl extends React.Component<IHealthCheckProps> {
   public static LABEL = 'Health Check';
 
-  public validate(): { [key: string]: string } {
-    return {};
+  public validate() {
+    return {} as FormikErrors<IAmazonClassicLoadBalancerUpsertCommand>;
```

Comment:
Potential improvement for the future: Make an interface for Wizard Pages to implement.  These validate functions can then be type checked betterlike